caiman/source_extraction/cnmf/cnmf.py

```diff
@@ -347,7 +347,9 @@ def fit_file(self, motion_correct=False, indices=None):
                 else:
                     b0 = np.ceil(np.max(np.abs(mc.shifts_rig))).astype(np.int)
                     self.estimates.shifts = mc.shifts_rig
-                b0 = 0 if self.params.get('motion', 'border_nan') is 'copy' else 0
+                # FIXME Huh?
```

Comment:
This line specifically looks like a bug to me. Is it intended?

Reply:
It specifies whether you want to exclude a number of pixels from the boundary. Right now it's always set to zero, but we might lift that in the future.

Reply:
Is _now_ a good time to allow it...? :)

Reply:
While I like that attitude of doing things instead of talking about them, this is a bit more complicated. `b0` is equal to maximum shift observed across *all* directions and a non-zero value will exclude the same number of pixels from all directions which can be grossly suboptimal. Ideally we need to enable the option of making this direction specific. This can require changes in many places of the code to function properly. So let's leave it as a placeholder for the time being.